### ISSUE TITLE
[Activity Wall] After privacy is changed, the user received the message about the privacy change. We should use .rtmedia-success class like we add it for other successful message.

### ISSUE THREAD

**Issue by kjnanda** [rtCamp/rtMedia#1479]
Please refer this visual - 

https://drive.google.com/file/d/1TZaeevEYE7q6VTiktrawLF0JT4olvF0x/view

-----

```
Test Environment:

WP - 5.3.0
rtMedia - 4.5.9
BuddyPress - 5.0..0 [Npuveau template]
theme - 2020
```

**Comment by dhavalkasavala** [comment]
@kjnanda - https://prnt.sc/q4ndul

**Comment by pooja-muchandikar** [comment]
https://github.com/rtCamp/rtMedia/pull/1491#issuecomment-930954216

**Comment by Pradeep1308** [comment]
PR Merged #1491 , hence closing it here.
